# Explore Errors Task

# Create a NameError
# nameerrors value is not referenced before being called

def createNameError():
	return nameerrors




#createNameError()